Userland/Shell/Shell.h

```diff
@@ -413,6 +414,10 @@ class Shell : public Core::Object {
     Shell();
     virtual ~Shell() override;
 
+    void destruct();
+    void default_construct();
+    void non_default_construct(bool attempt_interactive);
```

Comment:
Instead of these random methods, please leave the code in the constructors and the destructor...